src/pkg/cli/configList.go

```diff
@@ -24,7 +24,13 @@ func ConfigList(ctx context.Context, loader client.Loader, provider client.Provi
 		return err
 	}
 
-	configNames := make([]PrintConfig, 0, len(config.Names))
+	numConfigs := len(config.Names)
+	if numConfigs == 0 {
+		_, err := term.Warnf("No configs found")
+		return err
+	}
+
+	configNames := make([]PrintConfig, 0, numConfigs)
 	for _, c := range config.Names {
```

Comment:
this kind of loop, where an array is initialized from another array 1:1 without filtering, should just use regular indexing and not `append`

Reply:
i wonder if we can lint this. it's going to keep coming up.